doc/sphinx/source/tutorials/polarized_fits.rst

```diff
@@ -33,6 +33,45 @@ as a boundary condition while ``n_std`` specifies the shift in terms of the stan
 deviation to be applied to the PDF central values. If ``n_std=0.0`` then the
 PDF central values will be used to constrain their polarized counterparts.
 
+.. note::
+
+   This will also be the PDF set to be used to compute observable predictions
+   during the fit.
+
+Instead, the unpolarized PDF set to be used to compute observable predictions during
+data vs. theory comparisons or :math:`\chi^2` computations need to be specified under
+the ``datacuts`` entry:
```

Comment:
```suggestion
Instead, the unpolarized PDF set to be used to compute the `t0` covariance matrix need to be specified under the ``datacuts`` entry:
```

Note that you can also do `unpolarized_bc` in the outer namespace and let everything use it. The important point is that every polarized pdf (doesn't matter what is it used for) needs to have an `unpolarized_bc` associated in the validphys runcard.